ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-general.cfg

```diff
@@ -24,6 +24,21 @@ router general
       routes dynamic prefix-list DYNAMIC_TEST_PREFIX_LIST_2
       exit
    !
+   control-functions
+      code unit code1
+         function ACCEPT_ALL() {
+           return true;
+           }
+         EOF
+      code unit code2
+         function DENY_ALL() {
+           return true;
+           }
+         EOF
+
+      compile
+      commit
+   !
```

Comment:
```
router general
   control-functions
      code unit code1
         function ACCEPT_ALL() {
           return true;}
         EOF
      code unit code2
         function DENY_ALL() {
           return true;}
         EOF
      compile
      commit
   !
   exit
!
```

above format works with eapi but still need to find a way to support your format of config. Just FYI

Reply:
Okay!